src/main/java/org/mockito/MockSettings.java

```diff
@@ -249,10 +249,10 @@
      * @since 1.10.12
```

Comment:
Let's also update the description in main [Mockito Javadoc](https://static.javadoc.io/org.mockito/mockito-core/2.7.5/org/mockito/Mockito.html#spying_abstract_classes).